project1/css/index.css

```diff
@@ -28,6 +28,93 @@ article, aside, details, figcaption, figure,
 footer, header, hgroup, menu, nav, section {
 	display: block;
 }
+
+.container header{
```

Comment:
Not too important, but if you are interested in the "standards" of formatting CSS, it'd have a space after the selector, so instead of `.container header{` it'd be `.container header {` but that's not the most important thing in the world. It's most important that you're just consistent with your styling, more than that you follow any certain other style.